core/src/cluster_nodes.rs

```diff
@@ -480,11 +484,46 @@ pub fn make_test_cluster<R: Rng>(
     (nodes, stakes, cluster_info)
 }
 
+pub(crate) fn get_data_plane_fanout(shred_slot: Slot, root_bank: &Bank) -> usize {
+    if !enable_turbine_fanout_experiments(shred_slot, root_bank) {
+        return DATA_PLANE_FANOUT;
+    }
+    // Allocate ~2% of slots to fanout experiment.
+    match shred_slot % 359 {
+        11 => 64,
+        61 => 768,
```

Comment:
In Discord, we were discussing fanout == num validators (~2k). Would it make sense to have a higher max fanout option here?

Reply:
For now, I am a bit worried to go that high because of the egress burst.
We can follow up with another round of experiment with larger fanouts if it is shown that larger fanouts are better.